src/binaryen-c.h

```diff
@@ -103,6 +103,7 @@ typedef void* BinaryenFunctionTypeRef;
 // Add a new function type. This is thread-safe.
 // Note: name can be NULL, in which case we auto-generate a name
 BinaryenFunctionTypeRef BinaryenAddFunctionType(BinaryenModuleRef module, const char* name, BinaryenType result, BinaryenType* paramTypes, BinaryenIndex numParams);
+BinaryenFunctionTypeRef BinaryenGetFunctionType(BinaryenModuleRef module, BinaryenType result, BinaryenType* paramTypes, BinaryenIndex numParams);
```

Comment:
I am not sure how I feel about adding this API call. One issue is that we may eventually add getters by name, so this one might collide with the natural name for that one. Perhaps `BinaryenGetStructuralFunctionType`?

Another issue is that while this seems like a nice utility, it could be done by the user of the library that is creating the function types? Generally the binaryen API has been kept pretty minimal, and this feels like an "extra" - it's not strictly necessary, in other words. Another option to adding or not adding this might be to add it in a new section of the API (utilities? helpers?).